src/discovery/Akka.Discovery.AwsApi/README.md

```diff
@@ -66,37 +66,53 @@ akka.discovery {
 }
 ```
 
+## Using Discovery Together with Akka.Management and Cluster.Bootstrap
+All discovery plugins are designed to work with Cluster.Bootstrap to provide an automated way to form a cluster that is not based
+on hard wired seeds configuration. Some HOCON configuration is needed to make discovery work with Cluster.Bootstrap:
+
+```
+akka.discovery.method = aws-api-ec2-tag-based
+akka.management.http.routes = {
+    cluster-bootstrap = "Akka.Management.Cluster.Bootstrap.ClusterBootstrapProvider, Akka.Management.Cluster.Bootstrap"
+}
+```
+
+You then start the cluster bootstrapping process by calling:
```

Comment:
We need an end to end example somewhere that shows:

- Creation of `ActorSystem`
- Bootstrapping + Discovery calls
- A piece of output that demonstrates that the cluster has started successfully. A `Cluster.RegisterOnMemberUp` callback would do nicely.

Think of this as a piece of code that a user should be able to copy and paste into their application that will "just work" provided that their configuration and environments are correct. Please flesh this out to expand the context of what's going on at application startup.

Reply:
Also, you need to show right here which ports need to be open on each VM in order for this to work. Don't force the user to go digging around through the other documentation pages if you can pack it into a quick blurb here.